akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerMTSpec.scala

```diff
@@ -96,7 +100,7 @@ class CircuitBreakerMTSpec extends AkkaSpec {
       breaker.withCircuitBreaker(Future("succeed"))
       Await.ready(closedLatch, 5.seconds.dilated)
 
-      val futures = testCallsWithBreaker()
+      val futures = testCallsWithBreaker(breaker)
```

Comment:
same here, why 100 calls if at that stage the CB should be closed and we just want to confirm the call is passing? One call would suffice, no?

Reply:
I'm not sure but the whole test is about stressing it with MT=multi-threaded usage